core/grant/errors.go

```diff
@@ -4,6 +4,7 @@ import "errors"
 
 var (
 	ErrEmptyIDParam        = errors.New("grant id can't be empty")
+	ErrInvalidRequest      = errors.New("invalid request")
```

Comment:
what does invalid request represent?

Reply:
making it generic, usage can be `fmt.Errorf("%w: missing something from request", ErrInvalidRequest)`